src/Appium.Net/Appium/AppiumCommand.cs

```diff
@@ -164,8 +195,10 @@ public class AppiumCommand
                 "/session/{sessionId}/orientation"),
             new AppiumCommand(HttpCommandInfo.PostCommand, AppiumDriverCommand.SetOrientation,
```

Comment:
This should stay?

Reply:
No, these endpoints will remain https://github.com/appium/appium/blob/appium3/packages/base-driver/lib/protocol/routes.js